torch/_inductor/codegen/wrapper.py

```diff
@@ -445,6 +448,16 @@ def codegen_input_size_asserts(self):
             stride = self.codegen_shape_tuple(buf.get_stride())
             self.prefix.writeline(f"assert_size_stride({name}, {size}, {stride})")
 
+    def codegen_input_nan_asserts(self):
```

Comment:
Is the nancheck stuff part of this PR or a differnt PR?

Reply:
The nan checker for buffers used by generated triton kernels is added in an earlier PR. But that PR does not cover graph inputs. I just added that here for convenience since I've been debugging an nan issue caused by an earlier version of this PR.

Should I move this to a separate PR?

Reply:
Yeah, might be cleaner as another PR.